src/backend/distributed/metadata/node_metadata.c

```diff
@@ -664,6 +666,19 @@ ActivateNode(char *nodeName, int nodePort)
 }
 
 
+/*
+ * SetLockTimeoutLocally sets the lock_timeout to the given value.
+ * This setting is local.
+ */
+static void
+SetLockTimeoutLocally(int32 lock_cooldown)
+{
```

Comment:
minor:

We could define this right after `master_update_node` because:

* it is the first static function used in `master_update_node`
* the only place this function used is `master_update_node`

and then we can move its declaration on top of other static func declarations in this file (as we sort func declarations according to their definition order)